wolox-training/ap-swift
# Start API session from Login View

## Summary ##
**Trello card**: https://trello.com/c/oBIzM5n0/13-vista-de-biblioteca

Implemented login to WBooks API from login view

## Test Plan ##

  * Open the app
  * Login with Google
  * The main tab bar view should appear after a successfull login.

## Screenshots ##

### iPhone 7
![login-view](https://cloud.githubusercontent.com/assets/25587882/23278503/885d7694-f9f0-11e6-9643-31699daa54ef.png)
![main-view](https://cloud.githubusercontent.com/assets/25587882/23278506/89e0802e-f9f0-11e6-9a60-d3deb2860d69.png)
